config/settings.go

```diff
@@ -76,11 +78,26 @@ type Settings struct {
 	RequestIDBackendHeader    string   `hcl:"request_id_backend_header,optional"`
 	RequestIDClientHeader     string   `hcl:"request_id_client_header,optional"`
 	SecureCookies             string   `hcl:"secure_cookies,optional"`
+	TLSDevProxy               List     `hcl:"https_dev_proxy,optional"`
 	XForwardedHost            bool     `hcl:"xfh,optional"`
 	AcceptForwardedURL        []string `hcl:"accept_forwarded_url,optional"`
 	AcceptForwarded           *AcceptForwarded
 }
 
+type List []string
+
+func (s *List) String() string {
+	return strings.Join(*s, ",")
+}
+
+func (s *List) Set(val string) error {
```

Comment:
This method needs no return value

Reply:
Not provided here but just implements flag.Value. I have updated the file with a related type check.